src/store/actions/drinkCategories.js

```diff
@@ -1,35 +1,23 @@
 import * as actions from './actionTypes';
 import axios from '../../services/order-service';
 
-const setFetchedTypes = (actionVal, payload) => {
-  return {
-    type: actionVal,
-    drinks: payload
-  };
-};
+const setFetchedTypes = (actionVal, payload) => ({
+  type: actionVal,
+  drinks: payload,
+});
 
-export const fetchDrinkTypes = () => {
-  return dispatch => {
-    axios
-      .get('https://www.thecocktaildb.com/api/json/v1/1/list.php?c=list')
-      .then(response => {
-        dispatch(setFetchedTypes(actions.GET_DRINK_TYPES, response.data.drinks));
-      })
-      .catch(error => {
-        dispatch(console.log(error));
-      });
-  };
+export const fetchDrinkTypes = () => (dispatch) => {
+  axios
+    .get('https://www.thecocktaildb.com/api/json/v1/1/list.php?c=list')
+    .then((response) => {
+      dispatch(setFetchedTypes(actions.GET_DRINK_TYPES, response.data.drinks));
+    });
 };
 
-export const fetchAlcLevel = () => {
-  return dispatch => {
-    axios
-      .get('https://www.thecocktaildb.com/api/json/v1/1/list.php?a=list')
-      .then(response => {
-        dispatch(setFetchedTypes(actions.GET_ALC_LEVEL, response.data.drinks));
-      })
-      .catch(error => {
-        dispatch(console.log(error));
-      });
-  };
+export const fetchAlcLevel = () => (dispatch) => {
+  axios
+    .get('https://www.thecocktaildb.com/api/json/v1/1/list.php?a=list')
```

Comment:
This URL i being used in different places, let's create a constants file for this